app/src/main/java/de/dhbw/corona_world_app/api/APIManager.java

```diff
@@ -0,0 +1,167 @@
+package de.dhbw.corona_world_app.api;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import de.dhbw.corona_world_app.Logger;
+import de.dhbw.corona_world_app.datastructure.Country;
+import de.dhbw.corona_world_app.datastructure.Criteria;
+import de.dhbw.corona_world_app.datastructure.ISOCountry;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class APIManager {
+
+    private boolean cacheEnabled;
+
+    private boolean longTermStorageEnabled;
+
+    private String heroURL = "https://coronavirus-19-api.herokuapp.com";
+    private Map<String, String> heroToPopMap = new HashMap<>();
+    private Map<String, String> popToNameMap = new HashMap<>();
+
+    public APIManager(boolean cacheEnabled, boolean longTermStorageEnabled){
+        this.cacheEnabled = cacheEnabled;
+        this.longTermStorageEnabled = longTermStorageEnabled;
+        this.setMapData();
+    }
+
+    //TODO add all Countries into <Name,Alias> Map
+    public List<Country> getDataWorld(){
+        Logger.logD("getDataWorld","Getting Data for every Country");
+        String url = heroURL;
+        url += "/countries";
+
+        List<Country> returnList = new ArrayList<>();
+        String apiReturn = createAPICall(url);
+
+        StringToCountryParser.parseFromHeroMultiCountry(apiReturn,returnList);
```

Comment:
Typ der Api lieber als Enum in eine allgemeine Funktion einfügen, um neue Apis einfach einzufügen